modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -16,7 +16,7 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Prerequisites
 
-* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in _Preparing AWS credentials_.
+* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in _Preparing AWS credentials_. If you plan to use two different clusters for backing up and restoring, you need to prepare AWS credentials (including `ROLE_ARN`) for each cluster.
```

Comment:
```suggestion
* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in "Preparing AWS credentials". If you plan to use two different clusters for backing up and restoring, you need to prepare AWS credentials, including `ROLE_ARN`, for each cluster.
```

Use quotation marks for citations per IBM style guide: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks#double-quotation-marks

Do not use parentheses in general text, per IBM SG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses

Reply:
@abrennan89 
Done